pkg/delete/delete.go

```diff
@@ -107,7 +106,7 @@ func deleteClassicConfig(ctx context.Context, client dtclient.Client, theApi api
 	for _, v := range values {
 		log.WithCtxFields(ctx).WithFields(field.Type(theApi.ID), field.F("value", v)).Debug("Deleting %v (%v).", v, targetApi)
 		if err := client.DeleteConfigById(theApi, v.Id); err != nil {
-			errors = append(errors, err)
+			errors = append(errors, fmt.Errorf("could not delete %s configuration %q (%s): %w", theApi.ID, v.Name, v.Id, err))
```

Comment:
Very confusing message imo
```suggestion
			errors = append(errors, fmt.Errorf(`could not delete configuration "%s/%s" with ID %q: %w`, theApi.ID, v.Name, v.Id, err))
```

Reply:
Addressed but not exactly as you suggested. Logging the ID in brackets is consistent with many other logs in deployment and clients.